pkgs/os-specific/linux/kernel/linux-rpi.nix

```
@@ -39,6 +39,9 @@ lib.overrideDerivation (buildLinux (args // {
     # The v7 defconfig has this set to '-v7' which screws up our modDirVersion.
     sed -i $buildRoot/.config -e 's/^CONFIG_LOCALVERSION=.*/CONFIG_LOCALVERSION=""/'
     sed -i $buildRoot/include/config/auto.conf -e 's/^CONFIG_LOCALVERSION=.*/CONFIG_LOCALVERSION=""/'
+    # vc4 crashes on load with CONFIG_KUNIT=m
+    # https://github.com/NixOS/nixos-hardware/issues/631#issuecomment-1585757552
```

```suggestion
    # https://github.com/raspberrypi/linux/issues/5568
```
IMO it's better to link to the upstream bug report, so that it's easy to check when this modification can be dropped later.